lib/matplotlib/rcsetup.py

```diff
@@ -242,13 +243,14 @@ def validate_fonttype(s):
 
 _validate_standard_backends = ValidateInStrings(
     'backend', all_backends, ignorecase=True)
+_auto_backend_sentinel = object()
```

Comment:
I'm curious as to why this used as the sentinel, instead of the more typical 'auto' or None.

Reply:
Because we don't want to expose this as a public API (... at this point yet, at least).